lib/kubeclient/common.rb

```diff
@@ -253,6 +240,29 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def get_pod_log(pod_name, namespace, container: nil, previous: false)
```

Comment:
Hi, 
What is the previous param standing for? didn't notice it above in the readme.